src/Tizen.NUI/src/internal/WebView/WebSettings.cs

```diff
@@ -102,6 +105,56 @@ public bool EnableWebSecurity
             }
         }
 
+        /// <summary>
+        /// Enable cache builer
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool CacheBuilderEnabled
+        {
+            get 
+            {
+                return enableCacheBuilder;
+            }
+            set 
+            {
+                enableCacheBuilder = value;
+                Interop.WebSettings.EnableCacheBuilder(SwigCPtr, value);
+            }
+        }
+
+        /// <summary>
+        /// Enable do not track
```

Comment:
What is `Do not track`?

Reply:
"Do not track " is a terminology in web area,  use to protect user privacy security.

Reply:
If so can you add the description in the summary?

Reply:
Done